lib/gcloud/pubsub/subscription.rb

```diff
@@ -476,11 +476,11 @@ def delay new_deadline, *messages
       # A hash that conforms to the following structure:
       #
       #   {
+      #     "etag"=>"CAE=",
```

Comment:
The `version` attribute is in the documentation, but not in the data being returned by the service. I decided to trust the service's behavior over the documentation so I left `version` out.